python/orca/src/bigdl/orca/learn/mpi/mpi_runner.py

```diff
@@ -85,7 +87,9 @@ def run(self, file, **kwargs):
             mpi_config.extend(["-genv", "OMP_NUM_THREADS={}".format(mpi_env["OMP_NUM_THREADS"])])
         if len(self.remote_hosts) > 0:
             mpi_config.extend(["-hosts", ",".join(self.hosts)])
-        cmd.extend(mpi_config)
+        if mpi_options:
+            mpi_config += mpi_options
```

Comment:
Will here be any duplicated options?

Reply:
If there're duplicated options, then the mpi_options will overwrite previous ones.